src/main/java/mate/academy/service/UserService.java

```diff
@@ -14,7 +14,12 @@ public class UserService {
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
+    public static User findByEmail(String email) {
```

Comment:
this method should not be static. generally, when said 

> implement **method**

it's meant to implement not static method, because static are functions, not methods

```suggestion
    public User findByEmail(String email) {
```